### PR TITLE
feat: add optional remote fib service

### PR DESCRIPTION
**Pull request by beeme1mr**
## This PR

- Adds an additional fib service that can be optional enabled via a feature flag
- Fib service requires basic auth
- Removed unused scripts

Signed-off-by: Michael Beemer <beeme1mr@users.noreply.github.com>

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 version: '3.8'
 services:
   demo:
-    image: ghcr.io/open-feature/playground:openfeature-v0.0.7 # x-release-please-version
+    # image: ghcr.io/open-feature/playground:app-v0.0.7 # x-release-please-version
```
Comment (beeme1mr) [inline docker-compose.yaml:4]: TODO: Use the built images once they're built

File: packages/fibonacci-service/src/main.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import { NestFactory } from '@nestjs/core';
+
+import { AppModule } from './app/app.module';
+
+async function bootstrap() {
+  const app = await NestFactory.create(AppModule);
+  const port = process.env.PORT || 30001;
+  await app.listen(port);
+  Logger.log(`🚀 Application is running on: http://localhost:${port}/`);
+}
```
Comment (toddbaert) [inline packages/fibonacci-service/src/main.ts:16]: Maybe change this to say this is the fib app just to help in debugging.

File: packages/fibonacci-service/src/app/app.service.spec.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { Test } from '@nestjs/testing';
+
+import { AppService } from './app.service';
+
+describe('AppService', () => {
```
Comment (toddbaert) [inline packages/fibonacci-service/src/app/app.service.spec.ts:5]: Might want to delete this low-value test, or update it.

File: packages/fibonacci-service/src/app/app.controller.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import { Controller, DefaultValuePipe, Get, ParseIntPipe, Query, UseGuards } from '@nestjs/common';
+import { getNthFibBinet } from '@openfeature/fibonacci';
+import { AuthGuard } from './auth.guard';
+
+@Controller()
+@UseGuards(new AuthGuard())
+export class AppController {
+  @Get('calculate')
+  async getFibonacci(@Query('num', new DefaultValuePipe(50), ParseIntPipe) num: number) {
```
Comment (toddbaert) [inline packages/fibonacci-service/src/app/app.controller.ts:9]: `50` might be high?

**Review by toddbaert — APPROVED** [review]

**Pull request merged**